dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23ExternalContextStartupShutdownTests.java

```diff
@@ -67,6 +68,7 @@ public static void tearDown() throws Exception {
      * @throws Exception
      */
     @Test
+    @SkipForRepeat(SkipForRepeat.EE9_FEATURES)
```

Comment:
If we're skipping this b/c it does not work lets get an issue opened for investigation.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/13888